rtgui/rtwindow.cc

```diff
@@ -282,6 +282,14 @@ RTWindow::RTWindow ()
         show_all ();
 
         bpanel->init (this);
+
+        if (!argv1.empty()) {
+            Thumbnail* thm = cacheMgr->getEntry(argv1);
```

Comment:
My comment was about the lines below not above, sorry. It's more about saving the `std::vector<Thumbnail *> entries(1, thm);` not the control flow (which I hope is fine). :smile: